tests/UnifiedSpecTests/Util.php

```diff
@@ -26,13 +34,113 @@
+            'estimatedDocumentCount' => ['session', 'maxTimeMS'],
+            'deleteMany' => ['filter', 'session', 'collation', 'hint'],
+            'deleteOne' => ['filter', 'session', 'collation', 'hint'],
+            'findOneAndDelete' => ['filter', 'session', 'typeMap', 'projection', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint', 'maxTimeMS', 'new', 'sort', 'update', 'upsert'],
+            'distinct' => ['fieldName', 'filter', 'session', 'typeMap', 'collation', 'maxTimeMS'],
+            'drop' => ['session', 'typeMap'],
+            'find' => ['filter', 'session', 'typeMap', 'allowDiskUse', 'allowPartialResults', 'batchSize', 'collation', 'comment', 'cursorType', 'hint', 'limit', 'max', 'maxAwaitTimeMS', 'maxScan', 'maxTimeMS', 'min', 'modifiers', 'noCursorTimeout', 'oplogReplay', 'projection', 'returnKey', 'showRecordId', 'skip', 'snapshot', 'sort'],
+            'findOne' => ['filter', 'session', 'typeMap', 'allowDiskUse', 'allowPartialResults', 'batchSize', 'collation', 'comment', 'cursorType', 'hint', 'max', 'maxAwaitTimeMS', 'maxScan', 'maxTimeMS', 'min', 'modifiers', 'noCursorTimeout', 'oplogReplay', 'projection', 'returnKey', 'showRecordId', 'skip', 'snapshot', 'sort'],
+            'findOneAndReplace' => ['returnDocument', 'filter', 'replacement', 'session', 'typeMap', 'projection', 'returnDocument', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint', 'maxTimeMS', 'new', 'remove', 'sort'],
+            'replaceOne' => ['filter', 'replacement', 'session', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint'],
+            'findOneAndUpdate' => ['returnDocument', 'filter', 'update', 'session', 'typeMap', 'upsert', 'projection', 'remove', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint', 'maxTimeMS', 'sort'],
+            'updateMany' => ['filter', 'update', 'session', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint'],
+            'updateOne' => ['filter', 'update', 'session', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint'],
+            'insertMany' => ['options', 'documents', 'session', 'ordered', 'bypassDocumentValidation'],
+            'insertOne' => ['document', 'session', 'bypassDocumentValidation'],
+            'listIndexes' => ['session', 'maxTimeMS'],
+            'mapReduce' => ['map', 'reduce', 'out', 'session', 'typeMap', 'bypassDocumentValidation', 'collation', 'finalize', 'jsMode', 'limit', 'maxTimeMS', 'query', 'scope', 'sort', 'verbose'],
+        ],
+        ChangeStream::class => [
+            'iterateUntilDocumentOrError' => [],
+        ],
+        Cursor::class => [
+            'close' => [],
+            'iterateUntilDocumentOrError' => [],
+        ],
+        Session::class => [
+            'abortTransaction' => [],
+            'commitTransaction' => [],
+            'endSession' => [],
+            'startTransaction' => [],
```

Comment:
```suggestion
           'startTransaction' => ['maxCommitTimeMS', 'readConcern', 'readPreference', 'writeConcern'],
```

I realize this is unintuitive, but the [`startTransaction` operation](https://github.com/mongodb/mongo-php-library/blob/1.11.0/tests/UnifiedSpecTests/Operation.php#L614) passes all of its arguments directly to [`MongoDB\Driver\Session::startTransaction()`](https://www.php.net/manual/en/mongodb-driver-session.starttransaction.php) and thus supports the same options of _that_ method.

None of the unified spec tests currently use this argument; however, existing legacy tests do and those will eventually be converted by [DRIVERS-1709](https://jira.mongodb.org/browse/DRIVERS-1709). Since these arguments are already handled by the Operation class already in anticipation of those tests being ported, we should validate them now.